src/net.rs

```diff
@@ -0,0 +1,261 @@
+use futures::stream::TryStreamExt;
+use ipnetwork::IpNetwork;
+use rtnetlink::{new_connection, Error, NetworkNamespace};
+
+const VETHS_NAME: [&'static str; 2] = ["poeth1", "poeth2"];
```

Comment:
do we create veth pair with these names for every request? because we call `net_prep` for every request and `prep_network_stack` will create new bridge and veth pair every time